skipruntime-ts/examples/sheet.ts

```diff
@@ -85,7 +85,7 @@ class Service implements SkipService {
     context: Context,
     inputCollections: Record<string, EagerCollection<string, TJSON>>,
   ): Record<string, EagerCollection<TJSON, TJSON>> {
-    const cells = inputCollections["cells"];
+    const cells = inputCollections["cells"]!;
```

Comment:
Change type of parameter `inputCollections` to `{ cells: ... }`?

Reply:
Done in new version.